<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(240, 240, 240);</string>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>log</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_fiter">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>筛选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>关键字</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>22</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_check">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>查历史日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_now">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>看今天</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
